runtime/lua/json.lua

```diff
@@ -0,0 +1,383 @@
+--
+-- json.lua
```

Comment:
I didn't want to use the API to do it, since some of the json parsing might be done in a different thread I think. Unless it can work in another thread?